main.rb

```diff
@@ -27,18 +27,3 @@ def choose_number(choice, app)
     app.list_all_rentals
   end
 end
```

Comment:
- [ ] Good job by trying to refactor your code. However, your doesn't behavior in the the same way as in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) provided for you. I think this is because there is no method to start your app. Please make sure you fix this and test your app before requesting a new review request.